File: Brank/paiza5.swift
import Foundation 

let n = Int(readLine()!)!
var array: [(String, Int)] = []
for _ in 0..<n {
    let line = readLine()!.split(separator: " ")
    let str = String(line[0])
    let num = Int(line[1])!
    array.append((str, num))
}

let sortedArray = array.sorted(by: { $0.1 < $1.1 })

for tuple in sortedArray {
    print(tuple.0)
}

File: Brank/paiza5.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let n = 0;
let data = [];

rl.on('line', (input) => {
  if (n === 0) {
    n = parseInt(input);
  } else {
    const [s, d] = input.split(' ');
    data.push({s, d: parseInt(d)});
  }
 
  if (data.length === n) {
    data.sort((a, b) => a.d - b.d);
    data.forEach(({s}) => console.log(s));
    rl.close();
  }
});